Tests/Core/EndpointTests.swift

```diff
@@ -27,59 +27,46 @@ final class EndpointTests: XCTestCase {
     }
 
     func testEnvironmentParameter() throws {
-        var configuration: Configuration = .init(appId: "1", appSecret: "2", domain: .production, projectId: "1")
-        var endpoint: Endpoint<Mock> = .init(path: "/apps/mock", method: .get(nil), configuration: configuration) { _ in
+        var endpoint: Endpoint<Mock> = .init(path: "/apps/mock", method: .get(nil)) { _ in
             Mock(name: "foobar")
         }
         XCTAssertEqual(endpoint.domain, .production)
-
-        configuration = .init(appId: "1", appSecret: "2", domain: .staging, projectId: "1")
-        endpoint = .init(path: "/apps/mock", method: .get(nil), configuration: configuration) { _ in
-            Mock(name: "foobar")
-        }
-        XCTAssertEqual(endpoint.domain, .staging)
-
-        configuration = .init(appId: "1", appSecret: "2", domain: .testing, projectId: "1")
-        endpoint = .init(path: "/apps/mock", method: .get(nil), configuration: configuration) { _ in
-            Mock(name: "foobar")
-        }
-        XCTAssertEqual(endpoint.domain, .testing)
     }
 
     func testPathParameter() throws {
-        let configuration: Configuration = .init(appId: "1", appSecret: "2", domain: .production, projectId: "1")
-        var endpoint: Endpoint<Mock> = .init(path: "/apps/mock", method: .get(nil), configuration: configuration) { _ in
+        let configuration: Configuration = .init(appId: "1", appSecret: "2", domain: .production)
```

Comment:
Hier und an neuen weiteren Stellen sagt der Compiler:
`Immutable value 'configuration' was never used; consider replacing with '_' or removing it`

Reply:
06eb32dac989a432515f00efe7da9f21d93de975